input/2D/relaxation/distributed/poly_mps_ic_onestep_parallel.i

```diff
@@ -0,0 +1,336 @@
+[Mesh]
+ [gmg]
+  type = DistributedRectilinearMeshGenerator
+  dim = 2
+  nx = 125
+  ny = 125
+  xmin = 0
+  xmax = 20000
+  ymin = 0
+  ymax = 20000
+  partition = square
+  []
+  parallel_type = distributed
```

Comment:
Do not need " parallel_type = distributed" because DistributedRectilinearMeshGenerator will set a distributed format automatically

Reply:
Removed.